launcher/commands/install.py

```diff
@@ -173,6 +173,18 @@ def _create_full_install_args() -> Dict:
 
     return arguments
 
+def replace_string_in_file(file_path, target_string, replacement_string):
+    # Read the contents of the file
+    with open(file_path, 'r') as file:
```

Comment:
file_path is a Path, so you can use `file_path.read_text()`

Reply:
ready